invokeai/frontend/web/src/features/nodes/components/sidePanel/NodeEditorPanelGroup.tsx

```diff
@@ -1,26 +1,51 @@
 import 'reactflow/dist/style.css';
 
-import { Flex } from '@invoke-ai/ui-library';
+import { Box, Flex } from '@invoke-ai/ui-library';
+import { useStore } from '@nanostores/react';
+import { createMemoizedSelector } from 'app/store/createMemoizedSelector';
 import { useAppSelector } from 'app/store/storeHooks';
-import { selectWorkflowMode } from 'features/nodes/store/workflowSlice';
+import { overlayScrollbarsParams } from 'common/components/OverlayScrollbars/constants';
+import { selectNodesSlice } from 'features/nodes/store/selectors';
+import { $isWorkflowListMenuIsOpen } from 'features/nodes/store/workflowListMenu';
+import { selectWorkflowMode, selectWorkflowSlice } from 'features/nodes/store/workflowSlice';
 import ResizeHandle from 'features/ui/components/tabs/ResizeHandle';
-import WorkflowLibraryButton from 'features/workflowLibrary/components/WorkflowLibraryButton';
+import { OverlayScrollbarsComponent } from 'overlayscrollbars-react';
 import type { CSSProperties } from 'react';
-import { memo, useCallback, useRef } from 'react';
+import { memo, useCallback, useEffect, useRef } from 'react';
 import type { ImperativePanelGroupHandle } from 'react-resizable-panels';
 import { Panel, PanelGroup } from 'react-resizable-panels';
 
 import InspectorPanel from './inspector/InspectorPanel';
 import { WorkflowViewMode } from './viewMode/WorkflowViewMode';
 import WorkflowPanel from './workflow/WorkflowPanel';
-import { WorkflowMenu } from './WorkflowMenu';
-import { WorkflowName } from './WorkflowName';
+import { WorkflowListMenu } from './WorkflowListMenu/WorkflowListMenu';
+import { WorkflowListMenuTrigger } from './WorkflowListMenu/WorkflowListMenuTrigger';
 
 const panelGroupStyles: CSSProperties = { height: '100%', width: '100%' };
 
+const overlayScrollbarsStyles: CSSProperties = {
+  height: '100%',
+  width: '100%',
+};
+
+export const selectCleanEditor = createMemoizedSelector([selectNodesSlice, selectWorkflowSlice], (nodes, workflow) => {
+  const noNodes = !nodes.nodes.length;
+  const isTouched = workflow.isTouched;
+  return noNodes && !isTouched;
+});
+
 const NodeEditorPanelGroup = () => {
   const mode = useAppSelector(selectWorkflowMode);
   const panelGroupRef = useRef<ImperativePanelGroupHandle>(null);
+  const isWorkflowListMenuOpen = useStore($isWorkflowListMenuIsOpen);
+
+  const isCleanEditor = useAppSelector(selectCleanEditor);
+
+  useEffect(() => {
+    if (isCleanEditor) {
+      $isWorkflowListMenuIsOpen.set(true);
+    }
+  }, [isCleanEditor]);
```

Comment:
There is an awkward sequence:
- Editing a workflow
- Delete all nodes
- Save it
- The menu opens bc `isCleanEditor === true`

Maybe the suggested change the selector would resolve this

Reply:
Yep that seems to have fixed it